kubectl-fdb/cmd/root.go

```diff
@@ -68,6 +71,7 @@ func NewRootCmd(streams genericclioptions.IOStreams) *cobra.Command {
 			return cmd.Help()
 		},
 	}
+	skipCommand = !usingLatestPluginVersion()
```

Comment:
Can we add this as a `PreRunE`? Is there any benefit in the current setup compared to using `PreRunE`?